TypeScript.py

```diff
@@ -469,19 +495,22 @@ def change_count(self, view):
 
     # called by Sublime when a view receives focus
     def on_activated(self, view):
-        info = self.getInfo(view)
-        if info:
-            info.lastCompletionLoc = None
-            # save cursor in case we need to read what was inserted
-            info.prevSel = copyRegionsStatic(view.sel())
-            # ask server for initial error diagnostics
-            self.refreshErrors(view, 200)
-            # set modified and selection idle timers, so we can read
-            # diagnostics and update
-            # status line
-            self.setOnIdleTimer(20)
-            self.setOnSelectionIdleTimer(20)
-            self.changeFocus = True
+        # logger.view_debug(view, "enter on_activated")
```

Comment:
This, and several other logging lines below, you don't need to comment out.  You already have them at log level at 'DEBUG'.  Just change the level to 'INFO' or higher if you won't want to see them.